src/System Application/Test/AppSource Gallery/src/AppSourceProductManagerTest.Codeunit.al

```diff
@@ -0,0 +1,470 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace System.Apps.AppSource.Test;
+
+using System.Apps.AppSource;
+using System.RestClient;
+using System.Environment.Configuration;
+using System.TestLibraries.Utilities;
+
+codeunit 135074 "AppSource Product Manager Test" implements "IAppSource Product Manager Dependencies"
```

Comment:
We split the implementation from the test code. Also, the test cannot access the internal interface so this won't work

Reply:
Next PR